Shared/Samples/Style point with picture marker symbols/StylePointWithPictureMarkerSymbolsView.swift

```diff
@@ -0,0 +1,84 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct StylePointWithPictureMarkerSymbolsView: View {
+    /// A graphics overlay to hold the picture marker symbol graphics.
+    @State private var graphicsOverlay: GraphicsOverlay = {
+        let graphics = [
+            makePictureMarkerSymbolFromImage(),
+            makePictureMarkerSymbolFromURL()
+        ]
+        return GraphicsOverlay(graphics: graphics)
+    }()
+    
+    /// A map with topographic basemap and centered on Harman's Cross in England.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISTopographic)
+        map.initialViewpoint = Viewpoint(
+            center: Point(x: -225166.5, y: 6551249, spatialReference: .webMercator),
+            scale: 1e5
+        )
+        return map
+    }()
+    
+    /// Create a picture marker symbol from an image in the project assets.
+    private static func makePictureMarkerSymbolFromImage() -> Graphic {
```

Comment:
```suggestion
    /// Create a picture marker symbol from an image in the project assets.
    /// - Returns: A picture marker symbol.
```

In Xcode you can add a 3-slash documentation template (Xcode Markup, or commonly referred as docstring) by Editor > Selection > Add Documentation. Shortcut is `cmd + option + /`.

We add docstring for all non-trivial methods and properties.